content/webapp/pages/exhibition-guide.tsx

```diff
@@ -447,6 +526,18 @@ const ExhibitionGuidePage: FC<Props> = props => {
             </Layout8>
           </Header>
           <Space v={{ size: 'xl', properties: ['margin-top'] }}>
+            {userPreferenceSet ? (
+              <p>
+                This exhibition has {exhibitionGuide.components.length} stops.
+                You have indicated a preference for this guide type, to select
+                another type click Change guide type. // TODO: implement delete
+                cookie on click of Change guide type
+              </p>
+            ) : (
+              <p>
+                This exhibition has {exhibitionGuide.components.length} stops.
```

Comment:
nit: let's move this outside the conditional.

Also, has Dom reviewed this copy? I'd suggest something like:

> This is the BSL guide, which you used previously, but you can also select [another type of guide].

"indicates a preference" is a bit unclear to users, and most of them will tap rather than click.

Reply:
It’s my copy off the cuff, totally up for changing